dev/run-tests-jenkins.py

```diff
@@ -181,8 +181,8 @@ def main():
     short_commit_hash = ghprb_actual_commit[0:7]
 
     # format: http://linux.die.net/man/1/timeout
-    # must be less than the timeout configured on Jenkins (currently 350m)
-    tests_timeout = "300m"
+    # must be less than the timeout configured on Jenkins (currently 400m)
+    tests_timeout = "340m"
```

Comment:
we're STILL seeing test timeouts.  let's bump this to 400m and i'll up the timeout in jenkins to 430m.

Reply:
Woah .. got it ..